src/Features/VisualBasic/Portable/OrderModifiers/VisualBasicOrderModifiersCodeFixProvider.vb

```diff
@@ -15,5 +16,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.OrderModifiers
                        VisualBasicCodeStyleOptions.PreferredModifierOrder,
                        VisualBasicOrderModifiersHelper.Instance)
         End Sub
+
+        Protected Overrides ReadOnly Property FixableCompilerErrorIds As ImmutableArray(Of String) =
+            ImmutableArray(Of String).Empty
```

Comment:
might warrant an explanation.

Reply:
I think it makes sense, and allows for future expansion if someone (@MaStr11?) finds VB compiler diagnostics that would be fixed in a similar fashion.